mitosheet/src/mito/components/taskpanes/Graph/GraphSetupTab.tsx

```diff
@@ -145,7 +145,22 @@ function GraphSetupTab(
                     sheetIndex={graphSheetIndex}
                     onChange={(newSheetIndex) => {
                         // Reset the graph params for the new sheet, but keep the graph type!
-                        const newSheetGraphParams = getDefaultGraphParams(props.mitoContainerRef, props.sheetDataArray, newSheetIndex, props.openGraph)
+                        const newSheetGraphParams = getDefaultGraphParams(
```

Comment:
I'm still getting this bug https://github.com/mito-ds/mito/issues/1151 .But it looks like you caught a similar bug here, so very nice! 

The bug I saw is this:

1. Import a dataframe with columns A and B
2. Create a graph with A on the x axis and B on the Y axis
3. Switch back to the data tab
4. Delete column A
5. Switch back to the graph tab
6. Get this error

```
  File "column_headers.py", line 233, in get_column_headers_by_ids
    return [
  File "column_headers.py", line 234, in <listcomp>
    self.column_id_to_column_header[sheet_index][column_id]
KeyError: 'annual_income'
```


Instead of erroring, we want to just return the same graph, just without column A.

Reply:
Oh interesting, I'm not getting that error

Reply:
Are you sure you have the most recent code?

Reply:
Oh no I am in JupyterLab, interesting!